### ISSUE TITLE
UPI (PhonePe,) wallet and integration not working..

### ISSUE THREAD

**Issue by Rajaram10** [razorpay/react-native-razorpay#388]
## Description
When PhonePe wallet is selected, it keeps on loading with bottom popup "Your payment is being processed". 

## Razorpay Package Version :
2.2.7. 

## Java and Gradle Version (android) :
Java : 1.8.0_301.
Gradle : 4.1.0.

### What you did:
Followed React native razorpay docs.

### *Note:
 I am facing this issue when there is an UPI app installed in the same workspace along with my react native app.

https://user-images.githubusercontent.com/66991229/152109300-885e31cc-fb70-4a46-ac53-3c8f7610d404.mp4

**Comment by sagarkacha-robosoft** [comment]
Any solution for this issue? I am also facing same issue right now.

**Comment by vivekshindhe** [comment]
@sagarkacha-robosoft 
Unable to reproduce this issue. Can you share the payload you are using? To make the back & fro for the details, you can also create a ticket with Razorpay directly on the dashboard.

**Comment by vivekshindhe** [comment]
@sagarkacha-robosoft is the issue resolved?

**Comment by sagarkacha-robosoft** [comment]
@vivekshindhe I am still looking into this issue. below is the payload that I am passing with the Checkout class object to open Razorpay screen.

{
  "name": "Payment",
  "hidden": {
    "email": true,
    "contact": true
  },
  "description": "discription",
  "config": {
    "display": {
      "hide": [
        {
          "method": "emi"
        },
        {
          "method": "cardless_emi"
        },
        {
          "method": "paylater"
        },
        {
          "method": "upi"
        }
      ]
    }
  },
  "theme.color": "#000000",
  "currency": "INR",
  "amount": "100",
  "order_id": "xxxx",
  "modal.handleback": true,
  "image": "https://www.google.com/images/branding/googlelogo/2x/googlelogo_light_color_272x92dp.png"
}

**Comment by vivekshindhe** [comment]
@sagarkacha-robosoft Not able to reproduce the issue even with the payload you are sending. Would it possible for you to share the API Key you use? Or the order_id you where you have seen this issue over on email? Please forward these details at vivek.shindhe@razorpay.com. These details will help us look into the issue better. 
Thanks.

**Comment by sagarkacha-robosoft** [comment]
@vivekshindhe Can you please let me know if is there any configuration in the console that we need to do for the Phonepe wallet?

**Comment by vivekshindhe** [comment]
@sagarkacha-robosoft I did have that thought. But if it's not enabled phonepe option itself will not be visible to the user.

**Comment by sagarkacha-robosoft** [comment]
@vivekshindhe Yes, but I am asking because the Android native demo sample key I am using in my project then it's working fine but not with my API key So I suspect there should be some config in the console for the Phonepe wallet. Given in the below document also says Phonepe wallet requires approval.

https://razorpay.com/docs/payments/payment-methods/#supported-wallets
